builtin/game/statbars.lua

```diff
@@ -4,8 +4,9 @@ local enable_damage = core.settings:get_bool("enable_damage")
 local health_bar_definition = {
 	hud_elem_type = "statbar",
 	position = { x=0.5, y=1 },
-	text = "heart.png",
+	text = "heart.png/heart_gone.png",
```

Comment:
I know this is extremely convenient, but no.
The two texture names need to be separate fields.